src/components/map/mapbox/layers/ClusterLayer.tsx

```diff
@@ -339,6 +456,14 @@ const ClusterLayer: FC<ClusterLayerProps> = ({
         unclusterPointLayerMouseLeaveEventHandler
       );
       map?.on("mouseleave", clusterLayer, clusterLayerMouseLeaveEventHandler);
+
+      map?.on("click", clusterLayer, clusterLayerMouseClickEventHandler);
```

Comment:
line 256 of this file, rename too , e.g `clusterLayerMouseClickEventHandler` to `onClusterClick`